yb-voyager/cmd/common.go

```diff
@@ -1076,3 +1108,26 @@ func updateExportSnapshotDataStatsInPayload(exportDataPayload *callhome.ExportDa
 		}
 	}
 }
+
+func sendCallhomePayloadAtIntervals(ctx context.Context) {
+	sendTicker := time.NewTicker(20 * time.Minute) //TODO: confirm if this is fine
```

Comment:
i feel 20mins is lot, 10-15mins is fine i think

Reply:
okay , can make it to 15 mins